test/e2e/docs/troubleshooting.md

```diff
@@ -66,7 +66,6 @@ Solution:
 
 ```
 PUPPETEER_SKIP_DOWNLOAD=true
```

Comment:
Are we able to remove this too yet?

Reply:
I don't actually know where Puppeteer is used in `wp-calypso` and in my local environment I've always sidestepped the issue by setting that env var to `true`.

I'll do a quick check and if it's not too big of a change, will include it in this PR.

Reply:
I think I might remember. It could be pulled in via `@wordpress/scripts` or some of the wordpress monorepo helpers

Reply:
(additionally, the version of Puppeteer that's in `yarn.lock` is too old to support arm64 macOS)

Reply:
In any case, let's leave it for now and we can revisit Puppeteer in another patch.